cli/tests/activate.bats

```diff
@@ -275,8 +257,6 @@ env_is_activated() {
 
 # bats test_tags=activate,activate:hook,activate:hook:bash
 @test "bash: activate runs hook only once in nested activation" {
-  "$FLOX_BIN" delete -f
-  "$FLOX_BIN" init
```

Comment:
(non-blocking) Are these removed because `project_setup` should always gives us a fresh enviroment for each test case?

Reply:
Yep - I'm not really sure why they were added in the first place